pkg/eventprocessor/impl/executor/impl/txnscope_runsql_customfuncs_test.go

```diff
@@ -99,12 +92,13 @@ func TestCustomFunctionsWriteQuery(t *testing.T) {
 				ctx := context.Background()
 				ex, dbURI := test.newExecutorWithTable()
 
-				bs, err := ex.NewBlockScope(ctx, 0)
+				bs, err := ex.NewBlockScope(ctx, 1)
 				require.NoError(t, err)
 
 				txnHash, res, err := execTxnWithRunSQLEvents(t, bs, []string{test.query})
 				if test.mustFail {
-					require.Error(t, err)
+					require.NotNil(t, res.Error)
```

Comment:
Changed this because the errors are actually syntax errors and come through the `res`.